src/redis_string.cc

```diff
@@ -6,6 +6,34 @@
 
 namespace Redis {
 
+std::vector<rocksdb::Status> String::getRawValues(
+    const std::vector<Slice> &keys,
+    std::vector<std::string> *raw_values) {
+  raw_values->clear();
+
+  rocksdb::ReadOptions read_options;
+  LatestSnapShot ss(db_);
+  read_options.snapshot = ss.GetSnapShot();
+  std::vector<rocksdb::ColumnFamilyHandle*> cfs(keys.size(), metadata_cf_handle_);
+  auto statuses = db_->MultiGet(read_options, cfs, keys, raw_values);
+  for (size_t i = 0; i < keys.size(); i++) {
+    if (!statuses[i].ok()) continue;
+    Metadata metadata(kRedisNone, false);
+    metadata.Decode((*raw_values)[i]);
+    if (metadata.Expired()) {
+      (*raw_values)[i].clear();
+      statuses[i] = rocksdb::Status::NotFound(kErrMsgKeyExpired);
+      continue;
+    }
+    if (metadata.Type() != kRedisString && metadata.size > 0) {
```

Comment:
in redis, mget don't return an error, just return `$-1`, i think we should return an error.

Reply:
Good catch, we only transfer the not found error to nil string which may behave the same with Redis.